File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/EMRGetLatestWarningStatusInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
    public class EMRGetLatestWarningStatusInfo
    {
        private String _clisProtocolId;
        private String _statusId;
        private String _codeName;
        private String _updated;
        private String _displayFlg;

        public String ClisProtocolId
        {
            get { return this._clisProtocolId; }
            set { this._clisProtocolId = value; }
        }

        public String StatusId
        {
            get { return this._statusId; }
            set { this._statusId = value; }
        }

        public String CodeName
        {
            get { return this._codeName; }
            set { this._codeName = value; }
        }

        public String Updated
        {
            get { return this._updated; }
            set { this._updated = value; }
        }

        public String DisplayFlg
        {
            get { return this._displayFlg; }
            set { this._displayFlg = value; }
        }

        public EMRGetLatestWarningStatusInfo() { }

        public EMRGetLatestWarningStatusInfo(String clisProtocolId, String statusId, String codeName, String updated, String displayFlg)
        {
            this._clisProtocolId = clisProtocolId;
            this._statusId = statusId;
            this._codeName = codeName;
            this._updated = updated;
            this._displayFlg = displayFlg;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/EMRGetLatestWarningStatusInfo.java
package nta.med.data.model.ihis.emr;

import java.util.Date;

/**
 * @author Tiepnm
 */
public class EMRGetLatestWarningStatusInfo {

    private Integer clisProtocolId;
    private String statusId;
    private String codeName;
    private Date updated;
    private String displayFlg;

    public EMRGetLatestWarningStatusInfo(Integer clisProtocolId, String statusId, String codeName, Date updated, String displayFlg) {

        this.clisProtocolId = clisProtocolId;
        this.statusId = statusId;
        this.codeName = codeName;
        this.updated = updated;
        this.displayFlg = displayFlg;
    }

    public Integer getClisProtocolId() {
        return clisProtocolId;
    }

    public void setClisProtocolId(Integer clisProtocolId) {
        this.clisProtocolId = clisProtocolId;
    }

    public String getStatusId() {
        return statusId;
    }

    public void setStatusId(String statusId) {
        this.statusId = statusId;
    }

    public String getCodeName() {
        return codeName;
    }

    public void setCodeName(String codeName) {
        this.codeName = codeName;
    }

    public Date getUpdated() {
        return updated;
    }

    public void setUpdated(Date updated) {
        this.updated = updated;
    }

    public String getDisplayFlg() {
        return displayFlg;
    }

    public void setDisplayFlg(String displayFlg) {
        this.displayFlg = displayFlg;
    }

}
